cypress/e2e/tests/migration/archetypes/crud.test.ts

```diff
@@ -119,6 +120,39 @@ describe(["@tier1"], "Archetype CRUD operations", () => {
         notExists(archetypeDuplicate.name);
     });
 
+    it("Discard archetype review", function () {
+        // Automates Polarion MTA-428
+
+        const archetype = new Archetype(
+            data.getRandomWord(8),
+            [tags[0].name],
+            [tags[1].name],
+            null,
+            stakeholders,
+            stakeholderGroups
+        );
+
+        cy.log(`Tags: ${archetype.criteriaTags[0]}`);
+
+        archetype.create();
+        checkSuccessAlert(
```

Comment:
Please check is success alert is tested in one of the crud tests . If it is covered in any one test then it is not required here.

Reply:
done